commands/proposal_submission.go

```diff
@@ -1092,6 +1092,44 @@ func checkNewRiskParameters(config *protoTypes.NewMarketConfiguration) Errors {
 	return errs
 }
 
+func checkSLAParams(config *protoTypes.LiquiditySLAParameters, parent string) Errors {
+	errs := NewErrors()
+	if config == nil {
+		return errs.FinalAddForProperty(fmt.Sprintf("%s.sla_params", parent), ErrIsRequired)
+	}
+
+	lppr, err := num.DecimalFromString(config.PriceRange)
+	if err != nil {
+		errs.AddForProperty(fmt.Sprintf("%s.price_range", parent), ErrIsNotValidNumber)
+	} else if lppr.IsNegative() || lppr.IsZero() {
+		errs.AddForProperty(fmt.Sprintf("%s.price_range", parent), ErrMustBePositive)
+	} else if lppr.GreaterThan(num.DecimalFromInt64(100)) {
+		errs.AddForProperty(fmt.Sprintf("%s.price_range", parent), ErrMustBeAtMost100)
+	}
+
+	committmentMinTimeFraction, err := num.DecimalFromString(config.CommittmentMinTimeFraction)
+	if err != nil {
+		errs.AddForProperty(fmt.Sprintf("%s.committmentMinTimeFraction", parent), ErrIsNotValidNumber)
```

Comment:
Not a big deal, but we usually name fields in snake_case in the errors.

Reply:
renamed